app/src/main/java/com/sdp/swiftwallet/WalletActivity.java

```diff
@@ -4,16 +4,84 @@
 import androidx.constraintlayout.widget.ConstraintLayout;
 
 import android.annotation.SuppressLint;
+import android.app.Activity;
 import android.os.Bundle;
+import android.util.Log;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
 import android.widget.LinearLayout;
+import android.widget.ScrollView;
 import android.widget.TextView;
 
 import com.sdp.cryptowalletapp.R;
 
+import java.security.KeyStore;
+
 public class WalletActivity extends AppCompatActivity {
```

Comment:
We did those activities before to try the implementations, but think now that we are moving to fragment screens !